server/services/personService.test.ts

```diff
@@ -36,7 +40,7 @@ describe('PersonService', () => {
   })
 
   describe('getPerson', () => {
-    describe('when the prisoner client finds the corresponding prisoner', () => {
+    describe('when the prisoner client search finds the corresponding prisoner', () => {
```

Comment:
Does this change help? The description here feels agnostic to the details of the endpoint ('search' feels like an endpoint detail)